core/src/main/java/org/apache/servicecomb/core/provider/producer/ProducerProviderManager.java

```diff
@@ -97,7 +97,12 @@ private void onClose() {
       }
 
       if (Closeable.class.isInstance(operationMeta.getExecutor())) {
-        IOUtils.closeQuietly((Closeable) operationMeta.getExecutor());
+        Closeable executor = (Closeable) operationMeta.getExecutor();
```

Comment:
seems they are doing the same thing
why need the change?

Reply:
IOUtils.closeQuietly() is deprecated in commons-io 2.6